src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -377,6 +360,23 @@ def get_allowed_issue_creation_detectors(project_id: str):
         if rate and rate > random.random():
             allowed_detectors.add(detector_type)
 
+    for detector_type in list(allowed_detectors):
```

Comment:
This is a tiny list (a few items) to iterate, just splitting it up from system options that allow issue creation so it's a bit more readable